### PR TITLE
pulp_docker 3.2.7

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

**Comment by evgeni** [comment]
@ipanova I don't see any requirements to point at nectar, but my gut tells me it should be there and bumped to 1.6.2?

**Comment by evgeni** [comment]
another q: do you even want a full beta cycle with this? or just a 3.2.7 directly?

**Comment by ipanova** [comment]
> another q: do you even want a full beta cycle with this? or just a 3.2.7 directly?

let's have 3.2.7 directly

File: docs/user-guide/release-notes/3.2.x.rst [hunk 1]
```diff
@@ -1,6 +1,17 @@
 3.2 Release Notes
 =================
 
+3.2.7
```
Comment (ipanova) [inline docs/user-guide/release-notes/3.2.x.rst:4]: thanks!

**Review by ipanova — APPROVED** [review]

**Pull request merged**